samples/cellular/modem_shell/src/link/link_shell.c

```diff
@@ -369,6 +370,31 @@ static const char link_modem_usage_str[] =
 	"\n"
 	"Several options can be given and they are run in the given order.";
 
+static const char link_pall_usage_str[] =
+	"Usage: link pall --read | --write [options] | --clear\n"
+	"Options:\n"
+	"  -r, --read,            Read a PLMN access list.\n"
+	"      --write,           Write a PLMN access list.\n"
```

Comment:
Should there be a short for write?
```suggestion
	"  -w, --write,           Write a PLMN access list.\n"
```

Reply:
I don't think it's supported. I copied read and write from other commands.

Reply:
ok